### DISCUSSION TITLE
Breakup of datasets used during lung nodule detector training

### DISCUSSION THREAD

**Discussion by Rajesh-ParaxialTech** [Project-MONAI/tutorials#1739] [General]
Hello 

The datasets used for training the MONAI lung nodule detector were  the LUNA16 (which has around 888 datasets and is a subset of LIDC-IDRI) dataset right ? May I know the approximate breakup used for training and testing ?

Thanks
Rajesh

**Comment by ericspod** [comment]
@mingxin-zheng @tangy5 may be able to help with this one, I assume this refers to [this notebook](https://github.com/Project-MONAI/tutorials/blob/main/monailabel/monailabel_monaibundle_3dslicer_lung_nodule_detection.ipynb).

**Comment by mingxin-zheng** [comment]
Thanks @Rajesh-ParaxialTech for the question. I would defer the question @Can-Zhao about the dataset splitting.

**Comment by KumoLiu** [comment]
Hi @Rajesh-ParaxialTech, you can refer to the data section in this tutorial: https://github.com/Project-MONAI/tutorials/tree/main/detection#1-data.

Hope it helps, thanks.
(move to discussion for now)

**Comment by junxiant** [comment]
Hello, I was also looking through the data split. I took fold 0 for an example, used a python script to count the length, and I was getting:

training: 534
validation: 67

Is it correct to assume that there are 534 scans and 67 scans used for training and validation respectively?
